website/client/src/components/snackbars/notification.vue

```diff
@@ -133,6 +148,10 @@
     color: #2995cd;
   }
 
+  .damage .icon {
+    color: #686274;
```

Comment:
I know this isn't used much in this component, but could you switch from the raw color value to the variable `$gray-100`? You'll have to import the colors variables at the top `@import '~@/assets/scss/colors.scss';`

Reply:
I'll change the other colors to use vars too in the next PR, where there will be more changes to the styling.